ir/base.def

```diff
@@ -232,6 +232,8 @@ class Annotations {
 /// Implemented by all objects that can have annotations
 interface IAnnotated {
     virtual Annotations getAnnotations() const = 0;
+    Annotation getAnnotation(cstring name) const override {
```

Comment:
While in theory you can have multiple annotations with the same name, it almost never happens, so the common case is a single annotation.   We should have a special function that returns a set of annotations to deal with the multiple case, but we don't as no existing code deals with it, instead just picking the first one.